lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -310,32 +315,32 @@ def initialize(name, file_path, location, comments, visibility, owner) # rubocop
         @owner = owner
       end
 
-      sig { abstract.returns(T::Array[Parameter]) }
-      def parameters; end
+      sig { abstract.returns(T::Array[Entry::Signature]) }
+      def signatures; end
 
       # Returns a string with the decorated names of the parameters of this member. E.g.: `(a, b = 1, c: 2)`
       sig { returns(String) }
       def decorated_parameters
-        "(#{parameters.map(&:decorated_name).join(", ")})"
+        "(#{T.must(signatures.first).parameters.map(&:decorated_name).join(", ")})"
       end
     end
 
     class Accessor < Member
       extend T::Sig
 
-      sig { override.returns(T::Array[Parameter]) }
-      def parameters
+      sig { override.returns(T::Array[Signature]) }
+      def signatures
```

Comment:
An accessor will only ever have one signature, but we think it's easier to generalize this so that we can handle `Accessor` and `Method` in a similar way.